src/utils/check-pr/check-pr.py

```diff
@@ -189,6 +189,7 @@ def deploy(self, filename: str) -> None:
         subprocess.check_call(f"python -mvenv {venv}", shell=True)
         pip = venv_path(venv, "pip")
         py = venv_path(venv, "python")
+        config = os.getcwd() + "/config.json"
```

Comment:
We should build paths with functions like [`os.path.join()`](https://docs.python.org/3/library/os.path.html#os.path.join), not string concat.